admin/nodes/serializers.py

```diff
@@ -0,0 +1,25 @@
+from website.project.model import User
+from website.util.permissions import reduce_permissions
+
+from admin.users.serializers import serialize_simple_node
+
+
+def serialize_node(node):
+    user_list = {key: reduce_permissions(value) for key, value in node.permissions.iteritems()}
```

Comment:
Minor: it's a bit wasteful to go through two rounds of serialization to serialize contributors. You are looping through the contributors here, then again in the `map` call a few lines down. You should be able to reduce this to one iteration.